clients/banking/src/components/AccountArea.tsx

```diff
@@ -345,13 +345,22 @@ export const AccountArea = ({ accountMembershipId }: Props) => {
   const membership = useMemo(
     () =>
       currentAccountMembership.map(accountMembership => {
-        const { canInitiatePayments, canManageBeneficiaries, canViewAccount, legalRepresentative } =
-          accountMembership;
+        const {
+          canInitiatePayments,
+          canManageBeneficiaries,
+          canViewAccount,
+          legalRepresentative,
+          canManageCards,
+        } = accountMembership;
 
         const membershipEnabled = accountMembership.statusInfo.status === "Enabled";
         const canManageAccountMembership =
           accountMembership.canManageAccountMembership && membershipEnabled;
-        const canAddCard = canViewAccount && canManageAccountMembership && canOrderVirtualCards;
+        const canAddCard =
+          canViewAccount &&
+          // [NC] FIXME: retrocompatibility during canManageCards implementation :
+          (isNullishOrEmpty(canManageCards) ? canManageAccountMembership : canManageCards) &&
```

Comment:
`canManageCards` seems to be a mandatory boolean on read, we can simply use `canViewAccount && canManageCards`, the API has already defaulted to `canManageAccountMembership`